src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -662,4 +663,9 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
                         Id())
         };
     }
+
+    winrt::hstring UpdateSelectionArgs::GenerateName() const
+    {
+        return L"";

```

Comment:
Yeah, this is a good discussion to have, actually. I think we should hide it from the command palette just because VS Code doesn't have it in its command palette. They have "Expand Selection" and "Shrink Selection" which is a bizarre feature (you should check it out if you're curious).

Honestly, I was even debating removing it as a customizable action in the first place because (like the "copy" action), this can _only_ be used if a selection is active. So we're be claiming _a lot_ of key chords here.

Reply:
> I think we should hide it from the command palette

Fine with me. Seems weird to have in the command palette. 

Hard question though: how will it look in the SUI then?

Reply:
So, there's actually a nasty bug in SUI that this'll make more apparent: if the key binding has no name, it will appear as a nameless key binding in SUI. 😢

Opening a new issue now.

Reply:
https://github.com/microsoft/terminal/issues/10981